planner/functional_dependency/fd_graph.go

```diff
@@ -17,12 +17,12 @@ type FDSet struct {
 	fdEdges []*fdEdge
 }
 
-// closureOf is to find closure of X with respect to F.
+// closureOf is to find strict fd closure of X with respect to F.
 // A -> B  =  colSet -> { resultIntSet }
 // eg: considering closure F: {A-> CD, B -> E}, and input is {AB}
 // res: AB -> {CDE} (AB is included in trivial FD)
 // The time complexity is O(n^2).
-func (s *FDSet) closureOf(colSet FastIntSet) FastIntSet {
+func (s *FDSet) closureOfStrict(colSet FastIntSet) FastIntSet {
```

Comment:
`closureOfFD`? We possibly would not calc the closure of the lax ones.

Reply:
Since there is closureOfEquivalence, so I choose this similar name >.<. 
FD is sounds like containing all of that?